### PR TITLE
chore: update playwright

### PR DESCRIPTION
**Pull request by DaryaLari**
JsonObject has been removed here: https://github.com/microsoft/playwright/commit/e047c478a4943467cda926ff646e279f4bd01df0

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -27508,6 +27551,412 @@
                 }
             }
         },
+        "node_modules/vite/node_modules/@esbuild/aix-ppc64": {
```
Comment (ogonkov) [inline package-lock.json:27554]: esbuild and its deps are duplicated

File: src/components/AvatarStack/__snapshots__/AvatarStack.visual.test.tsx-snapshots/AvatarStack-render-story-MoreButton-dark-webkit-linux.png
Comment (ogonkov) [inline src/components/AvatarStack/__snapshots__/AvatarStack.visual.test.tsx-snapshots/AvatarStack-render-story-MoreButton-dark-webkit-linux.png:1]: Why it is changed?

File: package-lock.json (hunk 1 above)
Comment (DaryaLari) [reply package-lock.json]: fixed

File: src/components/AvatarStack/__snapshots__/AvatarStack.visual.test.tsx-snapshots/AvatarStack-render-story-MoreButton-dark-webkit-linux.png
Comment (DaryaLari) [reply src/components/AvatarStack/__snapshots__/AvatarStack.visual.test.tsx-snapshots/AvatarStack-render-story-MoreButton-dark-webkit-linux.png:1]: I suppose, in new pw-version browser was updated

Here only snapshot height have been changed, and actually, webkit snapshot size became the same as in chromium (and it was less expected, that they were different before)

**Review by ogonkov — APPROVED** [review]

**Review by ValeraS — APPROVED** [review]

**Pull request merged**